src/widget/wtrackmenu.cpp

```diff
@@ -581,25 +576,41 @@ void WTrackMenu::updateMenus() {
     if (featureIsEnabled(Feature::Color)) {
         m_pColorPickerAction->setColorPalette(
                 ColorPaletteSettings(m_pConfig).getTrackColorPalette());
-
-        // Get color of first selected track
-        TrackPointer pTrack = trackPointers.at(0);
-        auto trackColor = pTrack->getColor();
-
-        // Check if all other selected tracks have the same color
-        bool multipleTrackColors = false;
-        for (const auto& pTrack : trackPointers) {
-            if (trackColor != pTrack->getColor()) {
-                trackColor = mixxx::RgbColor::nullopt();
-                multipleTrackColors = true;
-                break;
+        std::optional<mixxx::RgbColor> oneColor = std::nullopt;
+        if (m_pTrackModel) {
+            const int column =
+                    m_pTrackModel->fieldIndex(LIBRARYTABLE_COLOR);
+            oneColor = mixxx::RgbColor::fromQVariant(
+                    m_trackIndexList.first().sibling(
+                                                    m_trackIndexList.first().row(), column)
+                            .data());
```

Comment:
Is this really how clang-format sets the whitespace? This is strange. Maybe we should adjust our clang-format settings.

Reply:
Yes it is, auto-formatted. If you know how, please make a proposal. But I have already too much time tuning those settings.

Reply:
I also have better ways to spend my time than worrying about edge cases with autoformatted whitespace.

Reply:
Otherwise I have to place everything on a single line.

Reply:
I'll join the lines. Case closed ;)